update.sh

```diff
@@ -91,9 +91,10 @@ print_basemetadata() {
 	    EXTRA_JAVA_OPTS="" \
 	    OPENHAB_HTTP_PORT="8080" \
 	    OPENHAB_HTTPS_PORT="8443" \
-	    LC_ALL=en_US.UTF-8 \
-	    LANG=en_US.UTF-8 \
-	    LANGUAGE=en_US.UTF-8
+	    LC_ALL="en_US.UTF-8" \
+	    LANG="en_US.UTF-8" \
+	    LANGUAGE="en_US.UTF-8" \
+	    CRYPTO_POLICY="limited"
```

Comment:
Would a boolean like thing make more sense, it feels a little bit error prone?

Reply:
Maybe we can let something fail if its neither limited or unlimited?

Reply:
I also think that a boolean parameter is better.

Reply:
I've thought about using a boolean as well but then found out how Java handles this with policies so opted for using name and values closely related to this mechanism. It looks like Oracle and OpenJDK also allow for other policies besides just "limited" and "unlimited". 

When using a boolean the variable name should  also change. Any suggestions for this? It should be short, meaningful and independent of the JVM used in the container.

Reply:
OK, if Oracle and OpenJDK use the values "limited" and "unlimited" we should do that as well.

Reply:
> martinvw 9 hours ago  Member
Maybe we can let something fail if its neither limited or unlimited?

@wborn wdyt?

Reply:
I thought about that too but did not implement it to keep it simple also because there were no checks for the [other environment variables](https://github.com/openhab/openhab-docker#environment-variables).